products/redhat-openjdk.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Red Hat OpenJDK
+category: lang
+tags: java-distribution redhat
+iconSlug: openjdk
+permalink: /redhat-openjdk
+versionCommand: java -version
+releasePolicyLink: https://access.redhat.com/articles/1299013
+releaseDateColumn: true
+eolColumn: Support
+releaseLabel: "OpenJDK __RELEASE_CYCLE__"
+
+# Based on searching for "openjdk runtime environment" and
+# "openjdk devel" at
+# https://access.redhat.com/downloads/content/package-browser
+# This is not great, since it is quite impossible to match
+# identifiers to a release cycle.
+identifiers:
+    - purl: pkg:redhat/java-1.6.0-openjdk
+    - purl: pkg:redhat/java-1.7.0-openjdk
+    - purl: pkg:redhat/java-1.7.0-openjdk-headless
+    - purl: pkg:redhat/java-1.8.0-openjdk
+    - purl: pkg:redhat/java-1.8.0-openjdk-debug
+    - purl: pkg:redhat/java-1.8.0-openjdk-headless
+    - purl: pkg:redhat/java-1.8.0-openjdk-headless-debug
+    - purl: pkg:redhat/java-11-openjdk
+    - purl: pkg:redhat/java-11-openjdk-debug
+    - purl: pkg:redhat/java-11-openjdk-headless-debug
+    - purl: pkg:redhat/java-1.6.0-openjdk-devel
+    - purl: pkg:redhat/java-1.7.0-openjdk-devel
+    - purl: pkg:redhat/java-1.8.0-openjdk-devel
+    - purl: pkg:redhat/java-1.8.0-openjdk-devel-debug
+    - purl: pkg:redhat/java-11-openjdk-devel
+    - purl: pkg:redhat/java-11-openjdk-devel-debug
+
+releases:
+-   releaseCycle: "17"
+    releaseDate: 2021-05-18 # RHEL 8.4
+    eol: 2027-10-30
+    # https://access.redhat.com/downloads/content/java-17-openjdk/x86_64/package-latest
+    latest: "17.0.6.0.10-6"
+    latestReleaseDate: 2023-04-26
+    link: https://access.redhat.com/documentation/en-us/openjdk/17/html-single/release_notes_for_openjdk_17.0.6/index
+
+-   releaseCycle: "11"
+    releaseDate: 2018-10-30 # RHEL 7.6
+    eol: 2024-10-30
+    # https://access.redhat.com/downloads/content/java-11-openjdk/x86_64/package-latest
+    latest: "11.0.19.0.7-2"
+    latestReleaseDate: 2023-04-26
+
+# 8 is a tricky release since its not supported on RHEL6
+# So we mark it twice, once specifically for RHEL6
+-   releaseCycle: "8"
+    releaseLabel: "OpenJDK 8 (1.8) - Windows / RHEL 7+"
+    releaseDate: 2015-03-05 # RHEL 7.1
+    eol: 2026-11-30
+    # https://access.redhat.com/downloads/content/java-1.8.0-openjdk/x86_64/package-latest
+    latest: "1.8.0.372.b07-3"
+    latestReleaseDate: 2023-04-27
+
+-   releaseCycle: "rhel6-8"
```

Comment:
I understand why this was added, but I think just a mention in the description is enough.

Reply:
This might need a broader discussion, since I want to figure out how to best handle `identifiers` in such cases. Will remove from here and file a separate issue.